Upstream pull-request: https://github.com/MycroftAI/mycroft-skills-kit/pull/45

From 0c22d30464f162727bf1b4f50705354e4e9b309f Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Mon, 4 May 2020 16:51:34 +0200
Subject: [PATCH] Remove the typing dep

Typing is a backport for Python older than 3.5, and isn't required on
newer versions. Besides Python 3.4 and lower isn't support anymore by
Mycroft anyway
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 246a4e7..9973754 100644
--- a/setup.py
+++ b/setup.py
@@ -26,7 +26,7 @@
     version='0.3.15',  # Also update in msk/__init__.py
     packages=['msk', 'msk.actions'],
     package_data={'msk': ['licenses/*']},
-    install_requires=['GitPython>=3.0.5', 'typing', 'msm>=0.5.13', 'pygithub',
+    install_requires=['GitPython>=3.0.5', 'msm>=0.5.13', 'pygithub',
                       'requests', 'colorama'],
     url='https://github.com/MycroftAI/mycroft-skills-kit',
     license='Apache-2.0',
